Projects/18_Clustering/silhouette.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+from sklearn.metrics import silhouette_samples
+from sklearn.cluster import KMeans
+import matplotlib.cm as cm
+import numpy as np
+
+def mean_silhouette_scores(Xdata,range):
```

Comment:
@amirziai 
I added this function to estimate the mean silhouette scores. The best number of clusters is 3.